### PR TITLE
#3261 | http_event_request_timeout set the HTTP timeout

### PR DESCRIPTION
**Pull request by tmmi**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala [hunk 1]
```diff
@@ -33,10 +33,19 @@ trait HttpEventConfiguration extends ScallopConf {
     default = Some(10.seconds.toMillis)
   )
 
+  lazy val httpRequestTimeout = opt[Long]("http_event_request_timeout",
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala]: please rename to `httpEventRequestTimeout`

File: src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala [hunk 2]
```diff
@@ -33,10 +33,19 @@ trait HttpEventConfiguration extends ScallopConf {
...
+    default = Some(10.seconds.toMillis)
+  )
+
   private[this] def parseHttpEventEndpoints(str: String): List[String] =
     str.split(',').map(_.trim).toList
 
-  def slowConsumerTimeout: FiniteDuration = httpEventCallbackSlowConsumerTimeout().millis
+  def slowConsumerDuration: FiniteDuration = httpEventCallbackSlowConsumerTimeout().millis
+
+  def requestTimeout: Timeout = Timeout(httpRequestTimeout(), TimeUnit.MILLISECONDS)
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala]: please rename to `eventRequestTimeout`

**Comment by aquamatthias** [comment]
@tmmi thanks for this PR! 
I added small suggestions: since all conf traits are merged together, it would be better to have more specific names.

File: src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala (hunk 1 above)
Comment (tmmi) [inline src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala]: done

File: src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala (hunk 2 above)
Comment (tmmi) [inline src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala]: clear

**Comment by aquamatthias** [comment]
@tmmi Thanks! LGTM.

**Pull request merged**